hedera-node/test-clients/src/main/resource/contract/contracts/NumericContract/NumericContract.sol

```diff
@@ -0,0 +1,164 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.5.0 <0.9.0;
+pragma experimental ABIEncoderV2;
+
+contract NumericContract {
+
+    /*´:°•.°+.*•´.*:˚.°*.˚•´.°:°•.°•.*•´.*:˚.°*.˚•´.°:°•.°+.*•´.*:*/
+    /*               Non-static Simple HTS functions              */
+    /*.•°:°.´+˚.*°.˚:*.´•*.+°.•°:´*.´•*.•°.•°:°.´:•˚°.*°.˚:*.´+°.•*/
+    function approveRedirect(address token, address account, uint256 amount) public {
+        (bool success, bytes memory result) = address(token).call(abi.encodeWithSignature("approve(address,uint256)", account, amount));
+
+        if (success == false) {
+            revert();
+        }
+    }
+
+    function approve(address token, address account, uint256 amount) public {
+        (bool success, bytes memory result) = address(0x167).call(abi.encodeWithSignature("approve(address,address,uint256)", token, account, amount));
+
+        if (success == false) {
+            revert();
+        }
+    }
+
+    function approveNFT(address token, address account, uint256 amount) public {
+        (bool success, bytes memory result) = address(0x167).call(abi.encodeWithSignature("approveNFT(address,address,uint256)", token, account, amount));
+
+        if (success == false) {
+            revert();
+        }
+    }
+
+    function burnTokenV1(address _token, uint64 _amount, int64[] memory _serialNumbers) external
+    returns (int32 responseCode, uint64 newTotalSupply)
+    {
+        (bool success, bytes memory result) = address(0x167).call(abi.encodeWithSignature("burnToken(address,uint64,int64[])",
+            _token, _amount, _serialNumbers));
+        (responseCode, newTotalSupply) =
+            success
+                ? abi.decode(result, (int32, uint64))
+                : (int32(0), 0);
+        if (responseCode != 22) {
```

Comment:
Any chance of defining a named constant for response code `22`?  (_You_ know it's `SUCCESS`, and _I_ know it's `SUCCESS`, but does _everyone_ know its `SUCCESS`?)

Reply:
Yes, I agree. I've changed it.